Mergin/project_status_dialog.py

```diff
@@ -231,3 +238,18 @@ def validate_project(self):
         else:
             self.show_validation_results(results)
             self.btn_sync.setStyleSheet("background-color: #ffc800")
+
+    def reset_local_changes(self):
+        btn_reply = QMessageBox.question(
+            None,
+            "Remove local changes",
+            "Local changes will be reverted to previous uploaded version. Some files might be removed or modified! Do you want to proceed?",
+            QMessageBox.Yes | QMessageBox.No,
+            QMessageBox.Yes,
+        )
+        if btn_reply == QMessageBox.No:
```

Comment:
does this work correctly e.g. if user presses escape button? I would probably check for `if btn_reply != QMessageBox.Yes` for the peace of mind that we're not reverting changes accidentally :)